Utility.js

```diff
@@ -122,3 +122,10 @@ export function getErrorString(error) {
     return String(error);
   }
 }
+
+export function getTimezoneOffsetTime(time) {
```

Comment:
This function should only round the time down to the day, not do timezone conversions

Reply:
It's worth noting that `getTimezoneOffset()` is dependent on the client's current timezone (from what I can tell). So if the coach happens to be in another timezone while using the app (what if the golf team has a tournament in another state etc), that could potentially cause issues.

If we want to "hard code a timezone", e.g. Pacific Time Zone, see: https://stackoverflow.com/a/64262840

Reply:
Rename this to getTimezoneOffsetDate because we're rounding to date